iothub/device/src/Authentication/IotHubConnectionCredentials.cs

```diff
@@ -132,6 +132,13 @@ public IotHubConnectionCredentials(string iotHubConnectionString)
         /// <summary>
         /// The full chain of certificates from the one used to sign the client certificate to the one uploaded to the service.
         /// </summary>
+        /// <remarks>
+        /// If supplied, the certificate chain will be verified and sent to the service in the request by adding them to the current user
```

Comment:
User-facing doc comment to help users navigate how this is used and how they can opt-in or -out of it.